.github/workflows/release.yml

```diff
@@ -64,6 +64,24 @@ jobs:
     needs: [finalize-release]
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/checkout@v4
+        # Pull the latest version of the reference
+        # branch instead of the revision that triggered
+        # the workflow otherwise we won't get the commit
+        # created in the previous job and this next job
+        # will fail.
+        with:
+          ref: ${{ github.ref }}
+      - name: Configure Identity
+        # Commits from github-actions do not
+        # trigger other GitHub Actions. As a result,
+        # we push from Ionitron instead so actions
+        # run when merging the release branch
+        # back into main.
+        run: |
+          git config user.name ionitron
+          git config user.email hi@ionicframework.com
+        shell: bash
```

Comment:
Same issues as https://github.com/ionic-team/ionic-framework/commit/0860ea09515039da4d606cfdb3ae9998b5089d2f and https://github.com/ionic-team/ionic-framework/commit/535b8ed72486a1e94125c72b12d5b447b630f8c4